src/util/realisticTyping.js

```diff
@@ -6,6 +6,7 @@ const delays = {
   deleteRange: 200,
   moveCursor: 100,
   applyDiff: 100,
+  punctuationDelay: 150,
```

Comment:
```suggestion
  typePunctuation: 150,
```
perhaps for consistent naming (to distinguish actions `type`, `delete`, `move`)